api/src/database/database.py

```diff
@@ -40,11 +58,12 @@ def __new__(cls, *args, **kwargs):
                     cls.instance = object.__new__(cls)
         return cls.instance
 
-    def __init__(self):
+    def __init__(self, echo_sql=True):
```

Comment:
Set the default to `True` to keep the current behavior. We should consider set it to False to avoid unnecessary SQL logs.